src/flint/fmpq.jl

```diff
@@ -759,6 +759,18 @@ reconstruct(a::Integer, b::ZZRingElem) =  reconstruct(ZZRingElem(a), b)
 
 reconstruct(a::Integer, b::Integer) =  reconstruct(ZZRingElem(a), ZZRingElem(b))
 
+@doc raw"""
+   unsafe_reconstruct(a::ZZRingElem, b::ZZRingElem)
+Same as [`reconstruct`](@ref), but does not throw if the reconstruction fails.
+Returns a tuple (`success`, `n/d`), where `success` signals the success of reconstruction.
```

Comment:
```suggestion
Returns a tuple (`success`, `a/b`), where `success` signals the success of reconstruction.
```

Reply:
Note that `a` is also an argument of this function, which could perhaps make things confusing

Reply:
I thought that you mean this `a`. If not, I should possibly read again what this function does